### ISSUE TITLE
Symbols Browser missing - Codeintel will not start on Centos 6 or RHEL 6

### ISSUE THREAD

**Issue by GrahamStuart** [Komodo/KomodoEdit#3393]
The Symbols Browser tab is not available even if turned on. No database is created in .komodoide. 

pystderr.log shows codeintel process is started. Two errors are reported:
[2017-11-29 09:18:45,319] [ERROR] ko-fileicons: Error: Command failed: 
....
[2017-11-29 09:18:45,474] [ERROR] sdk/editor: Error: Command failed:
....
 and then codeintel/process OnClose is called.

### Steps to Reproduce

Install K 11.0.1 on Centos 6, 32-bit or 64-bit. SeLinux on or off. iptables on or off. Start Komodo.

### Expected results

Symbols browser should be available on Centos 6.

### Actual results

Symbols browser is not available because codeintel process does not start.

### Platform Information

Komodo IDE 11.0.1 
Centos/RHEL 6.9 patched to date

### Additional Information

[pystderr.log](https://github.com/Komodo/KomodoEdit/files/1514831/pystderr.log)
[pystdout.log](https://github.com/Komodo/KomodoEdit/files/1514832/pystdout.log)

You can find your error log via Help > Troubleshooting > View Log File
-->

**Comment by Naatan** [comment]
Can you please provide codeintel3.log after reproducing the issue

**Comment by GrahamStuart** [comment]
Nope. codeintel3.log is not created. That's consistent. Customer didn't have one either.

**Comment by GrahamStuart** [comment]
-rw-r--r--  1 qatest qatest      637 Nov 29 09:15 .imsettings.log
drwx------  3 qatest qatest     4096 Dec  7  2015 .kde
drwxrwxr-x  3 qatest qatest     4096 Nov 29 09:18 .komodoide
drwxrwxr-x  5 qatest qatest     4096 Nov 29 09:18 Komodo-IDE-11
drwxr-xr-x  4 qatest qatest     4096 Nov 23 12:31 Komodo-IDE-11.0.1-90797-linux-x86_64
-rw-rw-r--  1 qatest qatest 80624714 Nov 23 12:20 Komodo-IDE-11.0.1-90797-linux-x86_64.tar.gz
drwxr-xr-x  3 qatest qatest     4096 Oct 16  2014 .local

**Comment by Naatan** [comment]
Opened a bug on the CentOS bug tracker: https://bugs.centos.org/view.php?id=14207

**Comment by th3coop** [comment]
@Naatan did you have any luck with this?  Looks like we need to bump up supported OS in the docs

**Comment by Naatan** [comment]
I've not made any progress on this since my last post.
